.gitignore

```diff
@@ -11,3 +11,7 @@ linux-kselftest
 src/gerrit_git_dir
 src/index_files
 src/logs
+.idea/
```

Comment:
Nit: Could you make these changes in a separate PR?

In this specific case, these config changes are pretty simple, but generally I like to keep changes specific to one logical thing - it makes things easier if you need to do a revert or do a git-blame.